include/eve/arch/arm/sve/tags.hpp

```diff
@@ -39,16 +39,20 @@ namespace eve
   //================================================================================================
   // Dispatching tag for ARM SIMD implementation
   //================================================================================================
-  struct sve128_  : simd_api<simd_   , spy::fixed_sve_> {};
-  struct sve256_  : simd_api<simd_   , spy::fixed_sve_> {};
-  struct sve512_  : simd_api<simd_   , spy::fixed_sve_> {};
+  struct sve_     : simd_api<simd_   , spy::sve_> {};
+  struct sve_vls_ : simd_api<sve_    , spy::fixed_sve_> {};
```

Comment:
Here is my thinking on `sve_`, `sve_vls_`

* we need a common tag type
* we can for now use `sve_` and `sve_vls_` interchangeably - obviously we don't support an unsized sve_ but if we feel like this function is unsized, we can use `sve_` tag. 

Then if/when we actually do the unsized sve, we will have less work (hopefully).

Reply:
let's have (per ARM nomenclature) :
 - `sve_`  
 - `sve_vls_` inheriting from `sve_`  
 - `sve_vla_` inheriting from `sve_`
 
 ?

Reply:
vla_ is `lengh agnostic`? then `vls_` should inherit `vla_`.
I just named `sve_` length agnostic one.

Can we do that in the next pr if this is the only change.

Reply:
Sure, can wait next PR.
I think I agree with you on the vsl/ vsa thing.